plugins/debian/debian-plugin.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
The name of the plugin file is supposed to be `plugins/{PluginName}/{PluginName}.plugin.sh`, so could you rename the file as `debian{- => .}plugin.sh` if you will make this a plugin?